fss_check/task_schedule.cfg

```diff
@@ -36,27 +34,16 @@ alert       taldcroft@cfa.harvard.edu
 <task fss_check3>
       cron * * * * *
       check_cron * * * * *
-      exec daily_fss.py --out $ENV{SKA_DATA}/fss_check3 --start=-90
-      exec daily_fss.py --out $ENV{SKA_DATA}/fss_check3 --start=-360
-      exec /bin/cp -p $ENV{SKA_DATA}/fss_check3/fss_prim/* /data/mta4/www/ASPECT/fss_check3/fss_prim/
-      exec /bin/cp -p $ENV{SKA_DATA}/fss_check3/fss_sec/* /data/mta4/www/ASPECT/fss_check3/fss_sec/
-      exec /bin/cp -p $ENV{SKA_DATA}/fss_check3/fss_prim_hist/* /data/mta4/www/ASPECT/fss_check3/fss_prim_hist/
+      exec fss_check_daily_fss --out $ENV{SKA}/data/fss_check3/dev --email=aca_alert
```

Comment:
I think this won't work either if acdc.common makes this into aca_alert@head.cfa.harvard.edu which isn't an address that HEAD knows about:
```
550 5.1.1 <[aca_alert@head.cfa.harvard.edu](mailto:aca_alert@head.cfa.harvard.edu)>: Recipient address rejected: User unknown in local recipient table
```
I think it is just a google group at aca_alert@cfa.harvard.edu.  Though perhaps we could ask for the head mapping?

Reply:
Fixed (again).